roles/edpm_ovn_bgp_agent/tasks/configure.yml

```diff
@@ -52,13 +52,15 @@
 - name: Add OVS Manager
   block:
     - name: Check if OVS Manager already exists
+      become: true
```

Comment:
Does this need to run under root? And how is this relevant to the dir ownership?

Reply:
this needs `become: true` like PR 683 did here, right?
https://github.com/openstack-k8s-operators/edpm-ansible/pull/683/files#diff-07e04cba7b71b24ec5e57638feeb63e9b4f1b09e43c56aa73d8e132c64a887f7
All tasks executing ovs-vsctl commands were modified to use `become: true`